app/tests/spec/lib/fxa-client.js

```diff
@@ -246,9 +246,9 @@ function (chai, $, sinon, FxaClient, p, testHelpers, FxaClientWrapper, AuthError
 
       it('still shows success after max tries', function () {
         sinon.stub(realClient, 'recoveryEmailResendCode', function () {
```

Comment:
You should be able to remove all of these `max tries` related tests since that functionality is no longer implemented. No reason to keep this code around.